force_wfmanager/left_side_pane/workflow_tree.py

```diff
@@ -32,40 +35,22 @@
 
 # Actions!
 
-call_modelview_editable = 'handler.modelview_editable(object)'
-
-# For reference, in the enabled_when expression namespace, handler is
-# the WorkflowTree instance, object is the modelview for a particular node -
-# in this case an MCOModelView
-
-new_mco_action = Action(name='New MCO...', action='new_mco')
-delete_mco_action = Action(name='Delete', action='delete_mco')
-edit_mco_action = Action(name='Edit...', action='edit_mco',
-                         enabled_when=call_modelview_editable)
-
-new_notification_listener_action = Action(
-    name='New Notification Listener...',
-    action='new_notification_listener')
-delete_notification_listener_action = Action(
-    name='Delete',
-    action='delete_notification_listener')
-edit_notification_listener_action = Action(
-    name='Edit...',
-    action='edit_notification_listener',
-    enabled_when=call_modelview_editable)
-new_parameter_action = Action(name='New Parameter...', action='new_parameter')
-edit_parameter_action = Action(name='Edit...', action='edit_parameter',
-                               enabled_when=call_modelview_editable)
-delete_parameter_action = Action(name='Delete', action='delete_parameter')
 new_kpi_action = Action(name='New KPI...', action='new_kpi')
-delete_kpi_action = Action(name="Delete", action='delete_kpi')
 new_layer_action = Action(name="New Layer...", action='new_layer')
-delete_layer_action = Action(name='Delete', action='delete_layer')
-new_data_source_action = Action(name='New DataSource...',
-                                action='new_data_source')
-delete_data_source_action = Action(name='Delete', action='delete_data_source')
-edit_data_source_action = Action(name='Edit...', action='edit_data_source',
-                                 enabled_when=call_modelview_editable)
+
```

Comment:
The delete context menu items are retained, along with addition actions with only add generic items (KPIs, Execution Layers)